apps/web/src/translations/subjects.json

```diff
@@ -158,5 +158,27 @@
       "en": "Value",
       "fr": "Valeur"
     }
+  },
+  "downloadInfo": {
+    "download": {
+      "en": "Download",
+      "fr": "Télécharger"
+    },
+    "subjectText": {
+      "en": "{{form}} of Subject: {{id}}",
+      "fr": "{{form}} du Sujet : {{id}}"
```

Comment:
In other areas, we translate "subject" as client, and do not use title case in French. So could you please change this to `{{form}} du client : {{id}}`.